changelog.md

```diff
@@ -1,10 +1,21 @@
 
 
+## 3.3.1 / 2020-12-17
+
+### Fixed Issues
+
+- Note: [#3722](https://github.com/tracim/tracim/issues/3722)
+- Mention: [#3927](https://github.com/tracim/tracim/issues/3927)
```

Comment:
```suggestion
- Mention: 
    - [#3722](https://github.com/tracim/tracim/issues/3722)
    - [#3927](https://github.com/tracim/tracim/issues/3927)
```

Reply:
At present, we use this syntax to show issue like this

![Screenshot_2020-12-17 tracim tracim](https://user-images.githubusercontent.com/25848257/102493169-130d7d80-4073-11eb-96d8-45d9c202ff5f.png)

In futur, we can show issue with your idea to add issue description (not available for the moment). This change need to be done in all changelog.

Reply:
I just seen you add 3722 in mention and not in Note, why ? Is it more related to mention ?

Reply:
This way it's good too, my suggestion was more to put the issue as mention and not note. I think is more related.

Reply:
Ok I put issue in Mention. Thanks.